core/src/test/java/com/google/zxing/pdf417/encoder/PDF417EncoderTestCase.java

```diff
@@ -67,4 +69,12 @@ public void testEncodeByte() throws Exception {
     assertEquals("\u039f\u001A\u0385abcd", encoded);
   }
 
+  @Test
```

Comment:
Make this `@Test(expected=WriterException.class)` and you can remove the try-catch

Reply:
Nice